chemicstry/wasm_thread
# Thread tests

I wanted to understand how the changes in the refactor branch "broke" my code

So I made a minimal reproduce of the bug as an example, also having more example (more end-to-end) might be beneficial

How to test the difference:

```
- Build the example (wasm_thread_example) with `./wasm-pack.sh`
- Serve the example with `npx serve pkg`
- Go on Chrome
- Open console and see if logs appear (change branch in Cargo.toml (refactor vs main)
- Repeat on different branches
```

What should you see:

- On branch `refactor` the worker spawns & despawns instantly
- On branch `main` the worker spawns & does its counting
